pkg/kubectl/cmd/convert.go

```diff
@@ -59,6 +58,28 @@ var (
 		kubectl convert -f . | kubectl create -f -`))
 )
 
+// ConvertOptions have the data required to perform the convert operation
+type ConvertOptions struct {
+	genericclioptions.IOStreams
```

Comment:
Move this and `FilenameOptions` to the bottom, like we've started doing recently.